ansible/group_vars/all.yml

```diff
@@ -153,3 +153,104 @@ time_servers:
   - 0.pool.ntp.org
   - 1.pool.ntp.org
   - 2.pool.ntp.org
+
+
+###############################
+# CFME Workload and Worker Appliances to Create
+###############################
+cfme_appliances:
```

Comment:
I think we should either consolidate this into the same dictionary for appliance configuration or move the to an entirely separate vars file to avoid confusion.